cmd/buildctl/build.go

```diff
@@ -56,8 +63,13 @@ var buildCommand = cli.Command{
 			Usage: "Define frontend used for build",
 		},
 		cli.StringSliceFlag{
-			Name:  "frontend-opt",
-			Usage: "Define custom options for frontend",
+			Name:   "frontend-opt",
+			Usage:  "Define custom options for frontend, e.g. --frontend-opt target=foo --frontend-opt build-arg:foo=bar (DEPRECATED: use --frontend-opts <opt>=<optval>[,<opt>=<optval>])",
+			Hidden: true,
+		},
+		cli.StringFlag{
```

Comment:
Could we make it stringslice and concat the values

Reply:
Btw, what are your thoughts on just renaming it `opt` or `args`

Reply:
I preferred `opt` , but it turned to out to have compatibility issue? https://github.com/moby/buildkit/pull/807#issuecomment-466076197

Reply:
I meant just `opt(s)` or `args`. Without frontend. As non-frontend builds don't take opts/args anyway.